openapi3filter/validate_request_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"testing"
```

Comment:
What made you edit this file?

Reply:
I realized I had not made it compliant with go test requirements (Test fnction prefix and args). The result was that when I run the tests I could not track where the specific problem was among the tests

Reply:
Please revert changes to this file. 

See https://blog.golang.org/examples section "Larger examples".